changelog/@unreleased/pr-4929.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: There are now fewer `getLeaderTime` calls when calling `getRowsColumnRangeIterator`.
```

Comment:
nit: when reading the iterator that is returned by `getRowsColumnRangeIterator`.